packages/shared/src/components/modals/PushNotificationModal.tsx

```diff
@@ -27,19 +28,37 @@ function PushNotificationModal(modalProps: ModalProps): ReactElement {
       {...modalProps}
       kind={Modal.Kind.FlexibleTop}
       size={Modal.Size.Medium}
+      isDrawerOnMobile
+      drawerProps={{ className: { drawer: 'pb-4', close: 'mx-4' } }}
     >
       <Modal.Header />
-      <Modal.Body>
-        <Modal.Title>Enable Push Notifications</Modal.Title>
-        <Modal.Text className="text-center">
+      <img
```

Comment:
This is one of the parts where I thought it didn't seem right. Where we will be rendering a lot of hidden elements and display based on screen size.

Reply:
We can make a quick drawer wrapper component which will internally check Modal context and then render or not render depending on `isDrawer`. 

```js
const DrawerImg = (props) => {
    const modalCtx = useModalContext()

    if (modalCtx.isDrawer) {
        return null
    }

    return (
        <img {...props} />
    )
}
```

```js
<Modal>
    <DrawerImg />
</Modal>
```